tests/connection_e2e_fcr_test.go

```diff
@@ -10,12 +10,21 @@ import (
 func TestCloudRouterCreate(t *testing.T) {
 	// retryable errors in terraform testing.
 	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
-		TerraformDir: "../examples/fabric/v4/cloudRouterConnectivity/cloudRouter",
+		TerraformDir: "../examples/fabric/v4/cloudRouter",
```

Comment:
This test is running the same directory twice. Just needs to remove the redundant code for a single test on cloudRouter.

Reply:
Ack.